engine/worker_task_conrrent_processor.go

```diff
@@ -186,7 +187,7 @@ func (w *workerTaskConcurrentProcessor) processWaitUntilTask(
 	}
 	if checkHttpError(err, httpResp) {
 		err := composeHttpError(err, httpResp)
-		w.logger.Info("worker API return error", tag.Error(err))
+		//w.logger.Info("worker API return error", tag.Error(err))
```

Comment:
curious why this is commented out?

Reply:
just for debug, sometimes there are too many logs so that I can't find the error log, will revert this change